### PR TITLE
chore(redhat-argocd): update wrapper

### PR DESCRIPTION
**Pull request by CryptoRodeo**
## Description

- Version 1.11.0 of the redhat-argocd backstage plugin has been released.
  - This new version migrates the PatternFly dependencies to V6.
- The wrapper has been updated, as well as yarn.lock.

## Which issue(s) does this PR fix

- Jira: [RHTAPUI-8](https://issues.redhat.com/browse/RHTAPUI-8)

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by CryptoRodeo** [comment]
FYI @karthikjeeyar

File: dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "export-dynamic:clean": "run export-dynamic --clean"
   },
   "dependencies": {
-    "@backstage-community/plugin-redhat-argocd": "1.10.4",
+    "@backstage-community/plugin-redhat-argocd": "1.11.0",
```
Comment (karthikjeeyar) [inline dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json:42]: could you please update the `version` field in the top of this file too? That is the one which will be displayed in the UI, so lets keep that in-sync with dependancy version.

Comment (CryptoRodeo) [reply dynamic-plugins/wrappers/backstage-community-plugin-redhat-argocd/package.json:42]: Good catch! changed.

**Review by karthikjeeyar — APPROVED** [review]
/lgtm

**Review by christoph-jerolimov — APPROVED** [review]

**Pull request merged**